build/build.sh

```diff
@@ -94,7 +98,7 @@ if [ "${BUILD_PDF:-}" != "false" ] && [ -n "$DOCKER_EXISTS" ]; then
     --security-opt=seccomp:unconfined \
     arachnysdocker/athenapdf:2.16.0 \
     athenapdf \
-    --delay=2000 \
+    --delay=${MANUBOT_ATHENAPDF_DELAY:-1100} \
```

Comment:
What does the `-` indicate?

Reply:
From https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html

> `${parameter:-word}`:
If parameter is unset or null, the expansion of word is substituted. Otherwise, the value of parameter is substituted.

So `:-` gets interpreted as a single operator. This PR also uses `:=`:

> `${parameter:=word}`
If parameter is unset or null, the expansion of word is assigned to parameter. The value of parameter is then substituted. Positional parameters and special parameters may not be assigned to in this way.

Reply:
Answering my own question with https://unix.stackexchange.com/questions/122845/using-a-b-for-variable-assignment-in-scripts